### PR TITLE
Add factions to profile

### LINKED ISSUE #18: Idea: Show Faction & Level in profile
It kinda goes without saying

### PR DESCRIPTION
**Pull request by Lycanium-Z**
Should close #18

### CODE REVIEW

File: wavelength-plugins/profile.js [hunk 1]
```diff
@@ -315,7 +315,8 @@ exports.commands = {
 				profile += '&nbsp;<font color="#24678d"><b>Group:</b></font> ' + userGroup + ' ' + devCheck(username) + vipCheck(username) + '<br />';
 				profile += '&nbsp;<font color="#24678d"><b>Registered:</b></font> ' + regdate + '<br />';
 				profile += '&nbsp;<font color="#24678d"><b>' + global.currencyPlural + ':</b></font> ' + currency + '<br />';
-				profile += '&nbsp;<font color="#24678d"><strong>EXP Level:</strong></font> ' + WL.level(toId(username)) + '<br />';
+				profile += '&nbsp;<font color="#24678d"><strong>Faction:</strong></font> ' + WL.level(toId(username)) + '<br />';
+				profile += '&nbsp;<font color="#24678d"><strong>EXP Level:</strong></font> ' + WL.getFaction(toId(username)) + '<br />';
```
Comment (HoeenCoder) [inline wavelength-plugins/profile.js]: level is faction, and faction is level

lol

**Comment by HoeenCoder** [comment]
(testing is to see if it displays properly still)

**Comment by Lycanium-Z** [comment]
Ok

File: wavelength-plugins/profile.js (hunk 1 above)
Comment (Lycanium-Z) [reply wavelength-plugins/profile.js]: *facepalm*

File: wavelength-plugins/profile.js [hunk 2]
```diff
@@ -315,6 +315,7 @@ exports.commands = {
 				profile += '&nbsp;<font color="#24678d"><b>Group:</b></font> ' + userGroup + ' ' + devCheck(username) + vipCheck(username) + '<br />';
 				profile += '&nbsp;<font color="#24678d"><b>Registered:</b></font> ' + regdate + '<br />';
 				profile += '&nbsp;<font color="#24678d"><b>' + global.currencyPlural + ':</b></font> ' + currency + '<br />';
+				profile += '&nbsp;<font color="#24678d"><strong>Faction:</strong></font> ' + WL.getFaction(toId(username)) + '<br />';
```
Comment (HoeenCoder) [inline wavelength-plugins/profile.js]: (WL.getFaction(user) || 'None')

**Comment by HoeenCoder** [comment]
This should be prevented https://imgur.com/a/AvhK9

File: wavelength-plugins/profile.js (hunk 2 above)
Comment (DeathlyPlays) [reply wavelength-plugins/profile.js]: Personally I think this should function as an if statement and not show if it doesn't exist like friend code

Comment (HoeenCoder) [reply wavelength-plugins/profile.js]: Im fine with either

**Pull request merged**